cves/kernel/CVE-2013-2635.yml

```diff
@@ -1,393 +1,101 @@
-
-  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
-  read to anyone with some programming experience. We can always pull up the NVD
-  description later to get more technical.
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
-bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
+description: |
+  This vulnerability was due to a miscalculation of a buffer size
+  relative to the data actually allocated to that buffer. 32 bytes were
+  allocated to the mac address buffer, but only 6 bytes of the buffer were
+  filled. This means that 26 bytes would end up leaking, which could be accessed
+  by an external application.
 bounty:
   amt:
   announced:
   url:
 reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
-
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
-bugs: []
-fixes_instructions: |
-  Please put the commit hash in "commit" below.
-
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
-
-  Place any notes you would like to make in the notes field.
+bugs: ["https://bugzilla.redhat.com/show_bug.cgi?id=924690"]
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: 84d73cd3fb142bf1298a8c13fd4ca50fd2432372
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
-vcc_instructions: |
-  The vulnerability-contributing commits.
-
-  These are found by our tools by traversing the Git Blame history, where we
-  determine which commit(s) introduced the functionality.
-
-  Look up these VCC commits and verify that they are not simple refactorings,
-  and that they are, in fact introducing the vulnerability into the system.
-  Often, introducing the file or function is where the VCC is, but VCCs can be
-  anything.
-
-  Place any notes you would like to make in the notes field.
+  note: Manually confirmed
 vccs:
 - commit: 5f8444a3fa617076f8da51a3e8ecce01a5d7f738
   note: Discovered automatically by archeogit.
 - commit: ebc08a6f47ee76ecad8e9f26c26e6ec9b46ca659
   note: Discovered automatically by archeogit.
```

Comment:
I'm not sure if this is right, but on the curation website, one of the instructions was to change the note to Manually Confirmed once you checked the git commits